key_value_service_trust_model.md

```diff
@@ -0,0 +1,163 @@
+    1. _Note that the values being stored in the key/value service are loaded by adtechs and expected to be publicly visible to clients. These are therefore not confidential._
+3. The user’s IP address could be used for fingerprinting. This is mitigated by the logging requirements and TEE protections to allow only pre-approved code.
+    2. We also have the option to route the traffic to these servers through [Gnatcatcher](https://github.com/bslassey/ip-blindness) to mask the user’s IP address.
+
+### Side effects
+
+There are a number of ways that visible side effects of request handling are possible in general with servers. Here’s how we plan to mitigate those effects:
+
+1. _Monitoring metrics_ - Only noised aggregate metrics will be available for monitoring and alerting.  These will be aggregated to at least k size.
+   1. For example, counting the approximate number of failed requests in the past n minutes is likely to be fine but doing so at millisecond granularity is not.
+2. _Logging_ - No event-level logs will be written.
+   1. Event-level logging that normally happens from any shared libraries we might use will be disabled.
+3. _Outbound RPCs_ - These servers will make a small set of outbound RPCs that they initiate.
+   1. They’ll do so to each other for load balancing and sharding of data. Requests will only be made to other key/value servers that are part of this system and that have the same protections in place. Attestation checks will be chained together.
+   1. There are no outbound RPCs to other systems.
+4. _Inbound RPC responses_ - As in the [API explainer](https://github.com/WICG/turtledove/blob/main/FLEDGE_Key_Value_Server_API.md), there are two sets of APIs.
+   1. For the client device to read K/V data. The client will provide its own secure channel and user data is allowed to go back to the browser.
+   2. For adtech server operators to mutate K/V data. These are private APIs that are only available to the server operator. The K/V server will acknowledge success or failure but not send other responses.
+
+## Trusted Execution Environment
+
+A Trusted Execution Environment (TEE) is a combination of hardware and software mechanisms that allows for code to execute in isolation, not observable by any other process regardless of the credentials used. The code running in a TEE will be open sourced and audited to ensure proper operation. Only TEE instances running an approved version of the key/value service code will be able to decrypt lookup requests.
```

Comment:
```suggestion
## Trusted execution environment

A trusted execution environment (TEE) is a combination of hardware and software mechanisms that allows for code to execute in isolation, not observable by any other process regardless of the credentials used. The code running in a TEE will be open sourced and audited to ensure proper operation. Only TEE instances running an approved version of the key/value service code will be able to decrypt lookup requests.
```

Reply:
This is not correctly capitalized for the aggregation service explainer, and should be lower case as such